Mathlib/Data/Nat/Sqrt.lean

```diff
@@ -114,9 +114,10 @@ theorem sqrt_zero : sqrt 0 = 0 := rfl
 
 theorem sqrt_eq_zero {n : ℕ} : sqrt n = 0 ↔ n = 0 :=
   ⟨fun h =>
-    Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide, by
-    rintro rfl
-    simp⟩
+    Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
+    by
```

Comment:
TBH I'm still not entirely sure about how to indent tuples when multiline formatting is needed. Possibilities include using 1 space indent like this:
```
  ⟨fun h => Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
   by rintro rfl; simp⟩
```
or 2 space indent with an awkward interior space:
```
  ⟨ fun h => Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
    by rintro rfl; simp ⟩
```
or just live with the misalignment:
```
  ⟨fun h => Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
    by rintro rfl; simp⟩
```
but in this last case you should indent subsequent lines (if more line breaking is necessary) such that they don't line up with the `by`:
```
  ⟨fun h => 
      Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
    by rintro rfl; simp⟩
```

Reply:
I've gone with the third of your proposals for this one:
```lean
  ⟨fun h => 
      Nat.eq_zero_of_le_zero <| le_of_lt_succ <| (@sqrt_lt n 1).1 <| by rw [h]; decide,
    by rintro rfl; simp⟩
```